packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -346,6 +346,12 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
     }
   }
 
+  private _dismissOnTargetWindowBlur = (ev: FocusEvent) => {
+    if (ev.relatedTarget === null && !document.hasFocus()) {
```

Comment:
Will this be the correct `document`? Wondering if you need to use either the `getDocument` utility or `this._targetWindow.document`.